os/pm/pm_resume.c

```diff
@@ -108,6 +114,14 @@ int pm_resume(int domain_id)
 		set_errno(ERANGE);
 		goto errout;
 	}
+#ifdef CONFIG_PM_METRICS
+	g_pm_metrics_globals.resume_ticks.counts[domain_id]++;
```

Comment:
What information can we obtain by knowing how many times a PM API is called?

Reply:
This count helps to get information how frequently suspend and resume APIs get called. For LCD case it is necessary because, we will be using SW timer. And frequent hits on suspend/ resume API can helps us to know the power-intensive domain indirectly.